test.js

```diff
@@ -0,0 +1,70 @@
+var tests = {
+
+    	numbers: [1, 5, 66, 224, 6, 22, 4, 1, 6, 8, 9, 86, 3, 4, 32, 64, 10],
+
+    	/*
+	Given an integer array of size N, compute the sum of all even numbers in this array
+	*/
+	firstTest: function () {
```

Comment:
The first implementation was definitely more interesting but this one is the proper one.